app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/MessageComposer.kt

```diff
@@ -82,6 +85,7 @@ private fun MessageComposer(
     onSendButtonClicked: () -> Unit
 ) {
     val focusManager = LocalFocusManager.current
+    var additionalInfoDisplayed by remember { mutableStateOf(false) }
```

Comment:
to keep it consistent and clean, please try to move this value to MessageComposerInnerState and propagte this event to MessageComposer